xds/src/main/java/io/grpc/xds/WeightedRoundRobinLoadBalancer.java

```diff
@@ -87,6 +96,29 @@ public WeightedRoundRobinLoadBalancer(WrrHelper helper, Ticker ticker, Random ra
     this.timeService = checkNotNull(helper.getScheduledExecutorService(), "timeService");
     this.updateWeightTask = new UpdateWeightTask();
     this.sequence = new AtomicInteger(random.nextInt());
+
+    MetricInstrumentRegistry metricInstrumentRegistry
+        = MetricInstrumentRegistry.getDefaultRegistry();
+    this.rrFallbackCounter = new MetricLongCounter(helper.getMetricRecorder(),
```

Comment:
The metrics design purposefully avoids per-instance state. And the LongCounter seems to make tests white box without a strong need. You'd need a separate one of these classes for each type of metric. I think we'll want to avoid this MetricLongCounter class/pattern.

You can only register things once, so this would fail in real life. Needing `reset()` seems broken.

Reply:
I was first going to have a single MetricCounter and use a factory method to either get a long or double metric. But then I figured it would be ok to add a separate double type one since just have the two types. But this is now a moot point as I don't need to do any counting at all and I can remove the counter class.

The reset was just there to allow unit tests to start from a clean slate (I initially had multiple).